etl/steps/data/garden/rff/2025-05-19/world_carbon_pricing.meta.yml

```diff
@@ -0,0 +1,68 @@
+definitions:
+  common:
+    description_key:
+      - This data primarily focuses on economic instruments targeting carbon dioxide (CO₂) emissions.
+      - In some cases these instruments also cover other greenhouse gases. However, any pricing mechanisms that specifically target non-CO₂ gases (such as methane or nitrous oxide) are not included.
+      - A country is considered to have a carbon tax or emissions trading system if at least one IPCC sector or gas is covered by the instrument. These instruments do not need to cover all sectors within the economy for this to apply.
+  presentation:
+    topic_tags:
+    - CO2 & Greenhouse Gas Emissions
+
+dataset:
+  update_period_days: 365
+
+tables:
+  world_carbon_pricing:
+    title: World carbon pricing
+    variables:
+      ets:
+        title: Covered by emissions trading system
+        unit: ""
+        description_short: A binary value indicating whether the sector(-fuel) is covered by at least one emissions trading system.
+        processing_level: minor
+      ets_price:
+        title: ETS price
+        unit: Current local currency unit per ton of CO₂ equivalent
+        short_unit: current LCU / t CO₂e
+        description_short: Price of an emissions allowance in current local currency unit per ton of CO₂ equivalent.
+        display:
+          numDecimalPlaces: 2
+        processing_level: minor
+      sector_name:
+        title: IPCC sector name.
+        unit: ""
```

Comment:
description_short would be useful to expand on this a bit (if it's going to be used, probably not worth it if not)